test/albums.js

```diff
@@ -0,0 +1,56 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  expect = chai.expect,
+  nock = require('nock');
+
+describe('/albums GET', () => {
+  it('should list albums', done => {
+    const albumRequest = nock('https://jsonplaceholder.typicode.com/')
```

Comment:
if you want to make the tests a little bit cleaner, you can move the nock logic to another file like `test/support/mocks.js` and here only require the desired mock

Reply:
Changed